test/summary.jl

```diff
@@ -5,7 +5,7 @@ using Base.Test
 graph = Graph()
 sess = Session(graph)
 mktempdir() do tmpdir
-    writer = tf.summary.FileWriter(tmpdir)
+    writer = tf.summary.FileWriter(joinpath("$tmpdir/test"))
```

Comment:
I think you mean to write:
` writer = tf.summary.FileWriter(joinpath(tmpdir,"test"))`
or ` writer = tf.summary.FileWriter("$tmpdir/test"))`

but you mistakenly combined the two